### ISSUE TITLE
Number of simultaneous tasks should be configurable (in config.xml) or some other place.

### ISSUE THREAD

**Issue by bastiao** [bioinformatics-ua/dicoogle#140]

**Comment by Enet4** [comment]
What would happen if an operation attempts to reach that limit?

**Comment by bastiao** [comment]
https://github.com/bioinformatics-ua/dicoogle/blob/dev/dicoogle/src/main/java/pt/ua/dicoogle/taskManager/TaskManager.java#L39

That's what I want to be dynamic.
To be honest, I think that a ThreadPool per type of tasks could also improve it.

**Comment by Enet4** [comment]
Oh, so you would want to limit the number of threads dedicated for running tasks. It's pretty tricky to achieve that with our current model, where each plugin is in charge of spawning tasks that can be associated to any executor or thread.

**Comment by bastiao** [comment]
but the ones triggered by PluginController.

**Comment by Enet4** [comment]
Yeah, I took back what I said a few minutes ago. In that case we have a rendez-vous point.
